.editorconfig

```diff
@@ -11,6 +11,9 @@ trim_trailing_whitespace = true
 [*.md]
 indent_size = unset
 
+[*.cs]
```

Comment:
Was the change to a four-character indent intentional as part of this PR?
I'd prefer that .editorconfig change to be done in a separate PR (and applied to all the .cs files in one go), and keep this one focused on the test enhancements